File: app/src/androidTest/java/com/wordpress/lonelytripblog/funwithflags/FunWithFlagsTestRunner.java
package com.wordpress.lonelytripblog.funwithflags;

import android.app.Application;
import android.content.Context;
import androidx.test.runner.AndroidJUnitRunner;

public class FunWithFlagsTestRunner extends AndroidJUnitRunner {
    @Override
    public Application newApplication(ClassLoader cl, String className, Context context) throws InstantiationException, IllegalAccessException, ClassNotFoundException {
        return super.newApplication(cl, TestApp.class.getName(), context);
    }
}

File: app/src/androidTest/java/com/wordpress/lonelytripblog/funwithflags/FunWithFlagsTestRunner.kt
package com.wordpress.lonelytripblog.funwithflags

import android.app.Application
import android.content.Context
import androidx.test.runner.AndroidJUnitRunner

class FunWithFlagsTestRunner : AndroidJUnitRunner() {
    @Throws(InstantiationException::class, IllegalAccessException::class, ClassNotFoundException::class)
    override fun newApplication(cl: ClassLoader, className: String, context: Context): Application {
        return super.newApplication(cl, TestApp::class.java.name, context)
    }
}